recipes/imguizmo/all/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.4)
```

Comment:
I don't know if imguizmo is abandoned, but it would be great creating a PR to upstream providing cmake support.

Reply:
ImGui (and most of its sister projects)'s philosophy is to not provide any build systems, and written in a way such that you can just drop all the relevant files to your project and works with no extra setup. So I don't think such a request will go anywhere.

On a side note, I think issues should be filed if this CMakeLists.txt causes error in the future, because that violates the promise of "dropping files into project and done".

Reply:
Thanks for commenting. Yes, indeed we use a custom CMake file for imgui .